lsl_autobids/dataverse_dataset_create.py

```diff
@@ -2,15 +2,34 @@
 from pyDataverse.api import NativeApi
 from pyDataverse.models import Dataset
 from pyDataverse.utils import read_file
-import toml
-import json
-import tomllib
+import json 
 import os
-from globals import project_root, api_key,dataverse_base_url,parent_dataverse_name
-
+from config_globals import project_root, dataverse_base_url, api_key, parent_dataverse_name
+import logging
+from utils import read_toml_file, write_toml_file
 
+# Set up logging
+logger = logging.getLogger(__name__)
```

Comment:
I'm actually wondering if we should add the log files to the BIDS somewhere? Maybe a nice-to-have feature to document in an issue?